### PR TITLE
Uncertainpy recipe

### PR DESCRIPTION
**Pull request by volpatto**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by volpatto** [comment]
@conda-forge/python @conda-forge/staged-recipes Hi! I would like to kindly request a review and eventually the merge of this new recipe. It's a pure Python package, I think it's ready to merge, but please check it for me. Thanks!

File: recipes/uncertainpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    - uncertainpy.core
+    - uncertainpy.features
+    - uncertainpy.models
+    - uncertainpy.plotting
+    - uncertainpy.plotting.prettyplot
+    - uncertainpy.utils
+
+about:
+  home: https://github.com/simetenn/uncertainpy
+  license: GNU GPLv3
```
Comment (chrisburr) [inline recipes/uncertainpy/meta.yaml]: From https://spdx.org/licenses/:
```suggestion
  license: GPL-3.0
```

File: recipes/uncertainpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  license_family: GPL3
```
Comment (chrisburr) [inline recipes/uncertainpy/meta.yaml]:
```suggestion
  license_family: GPL
```

File: recipes/uncertainpy/meta.yaml (hunk 1 above)
Comment (volpatto) [reply recipes/uncertainpy/meta.yaml]: I didn't know about it... conda skeleton generated this way above. Should I relate it?

**Pull request merged**